lib/automation/report_runner.rb

```diff
@@ -138,7 +149,22 @@ def run!
 
             #{input}
           RAW
-          PostCreator.create!(@sender, raw: raw, topic_id: post.topic_id, skip_validations: true)
+            PostCreator.create!(@sender, raw: raw, topic_id: post.topic_id, skip_validations: true)
+          end
+        end
+
+        if @email_receivers.present?
+          @email_receivers.each do |to_address|
+            Email::Sender.new(
+              ::AiReportMailer.send_report(
+                to_address,
+                subject: @title,
+                body: result,
+                from_address: SiteSetting.notification_email,
```

Comment:
You shouldn't need this, the default is to use `notification_email`...I am pretty sure

https://github.com/discourse/discourse/blob/4c8bc34475d0c9dc22d790f5ef3b05570e97048f/lib/email/message_builder.rb#L283-L287